docs/guide/plugins/plugins.md

```diff
@@ -52,7 +52,7 @@ The Lua code file referenced in the `plugin.ini` file must define an `initUi`-fu
 
 - the `menu` name displayed in Xournal++'s menubar,
 - the `callback` function used when the plugin is called from the menu or via a keyboard accelerator,
-- the `accelerator` used to execute the plugin callback function. The format looks like `<Control>a` or `<Shift><Alt>F1` or `<Release>z` (for key release). See [GTK3 reference](https://developer.gnome.org/gtk3/stable/gtk3-Keyboard-Accelerators.html#gtk-accelerator-parse) for details and note that release key accelerators currently do not work, see [this issue](https://github.com/xournalpp/xournalpp/issues/2396). The accelerator can be omitted. Using accelerators without modifiers like (`<Alt>`, `<Control>`, `<Shift>`) is allowed, but will likely break the text tool.
+- the `accelerator` used to execute the plugin callback function. The format looks like `<Control>a` or `<Shift><Alt>F1` or `<Release>z` (for key release). See [GTK3 reference](https://docs.gtk.org/gtk3/func.accelerator_parse.html) for details and note that release key accelerators currently do not work, see [this issue](https://github.com/xournalpp/xournalpp/issues/2396). The accelerator can be omitted. Using accelerators without modifiers like (`<Alt>`, `<Control>`, `<Shift>`) is allowed, but will likely break the text tool.
```

Comment:
GTK3 documentation on accelerators seems to be broken due to the tags being parsed incorrectly. I opened an issue on their GitLab but I'm not sure how fast they fix it. The [GTK4 docs](https://docs.gtk.org/gtk4/func.accelerator_parse.html) seem to be fine, would that be an option?

Reply:
Yes, the GTK4 docs look fine. Interestingly they don't mention a Release key any more (and I think it never got implemented, reading through the code file that does the parsing). So `<Release>z` shouldn't be mentioned at all, issue [#2396](https://github.com/xournalpp/xournalpp/issues/2396) should be closed and not mentioned.

Reply:
I hope I understood you correctly. It links to the GTK4 docs now and the release-accelerator isn't mentioned anymore, is that alright?

Reply:
Yes, that's fine.